screener.config.js

```diff
@@ -41,38 +55,40 @@ const screenerConfig = {
   },
 
   // screenshot every example in maximized mode
-  states: glob
-    .sync('docs/src/examples/**/*.tsx', { ignore: ['**/index.tsx', '**/*.knobs.tsx'] })
-    .reduce((states, examplePath) => {
-      const {
-        name: exampleNameWithoutExtension,
-        base: exampleNameWithExtension,
-        dir: exampleDir,
-      } = path.parse(examplePath)
-
-      const rtl = exampleNameWithExtension.endsWith('.rtl.tsx')
-      const exampleUrl = _.kebabCase(exampleNameWithoutExtension)
-
-      states.push({
-        url: `http://${SCREENER_HOST_URL}/maximize/${exampleUrl}/${rtl}`,
-        name: exampleNameWithExtension,
-
-        // https://www.npmjs.com/package/screener-runner#testing-interactions
-        steps: getSteps(exampleDir, exampleNameWithoutExtension),
-      })
-
-      return states
-    }, []),
+  states: filteredPaths.reduce((states, examplePath) => {
+    const {
+      name: exampleNameWithoutExtension,
+      base: exampleNameWithExtension,
+      dir: exampleDir,
+    } = path.parse(examplePath)
+
+    const rtl = exampleNameWithExtension.endsWith('.rtl.tsx')
+    const exampleUrl = _.kebabCase(exampleNameWithoutExtension)
+
+    states.push({
+      url: `http://${SCREENER_HOST_URL}/maximize/${exampleUrl}/${rtl}`,
+      name: exampleNameWithExtension,
+
+      // https://www.npmjs.com/package/screener-runner#testing-interactions
+      steps: getSteps(exampleDir, exampleNameWithoutExtension),
+    })
+
+    return states
+  }, []),
 }
 
 function getSteps(dir, nameWithoutExtension) {
   const stepsSpecModulePath = `${dir}/${nameWithoutExtension}.steps.ts`
 
-  return fs.existsSync(stepsSpecModulePath)
-    ? require(`./${dir}/${nameWithoutExtension}.steps`)
-        .default.reduce((stepsAcc, steps) => steps(stepsAcc), new Steps())
-        .end()
-    : undefined
+  if (fs.existsSync(stepsSpecModulePath)) {
+    const stepsDefinition = require(`./${dir}/${nameWithoutExtension}.steps`).default
+
+    return stepsDefinition
+      .reduce((stepsAcc, steps) => {
+        return steps(stepsAcc, Keys).executeScript(`window.resetExternalLayout()`)
```

Comment:
just to clarify - is it intentional that `executeScript` is not introduced to  `ScreenerRunnerSteps` type?

Reply:
Missed it in type definitions, added 👍